todo/src/components/TodoList.js

```diff
@@ -1,10 +1,18 @@
 import React from 'react';
 import Todo from './Todo';
+import { connect } from 'react-redux';
 
 const TodoList = props => {
   return props.todos.map(todo => (
-    <Todo todo={todo} key={todo.id} toggleTodo={props.toggleTodo} />
+    <Todo todo={todo} key={todo.id} />
   ))
 }
 
-export default TodoList;
+
+const mapStateToProps = (state) => {
+    return {
+        ...state
```

Comment:
this is the only component that actually needs the list of todos from state, so this belongs here. what I'd recommend however, is always be as explicit as possible to make sure you're only pulling in what you need: because you only need the `todos` array from state here, set that as on a prop to be accessed by this component like `todos: state.todos`, which will then be referenced by `props.todos` anyway, but now you're _explicitly_ stating what pieces of state you're pulling in and mapping to your props instead of the entire state tree